csharp/RemotePartyFinder/Gatherer.cs

```diff
@@ -53,6 +47,7 @@ private void OnUpdate(IFramework framework1) {
         }
 
         this.UploadTimer.Restart();
+        List<UploadUrl> uploadUrls = Plugin.Configuration.UploadUrls.Select(x => x.Clone()).ToList();
```

Comment:
Is this clone to prevent concurrent modification errors?
If so, why not use a concurrent list?

Reply:
Yeah it is. There doesn't seem to be a direct equivalent of Lists in `System.Collections.Concurrent` so I didn't use it. Also to be frank, I'm not super experienced with C# and what it has to offer as a whole, so I referenced Dalamud's code for its third party repo list. I figured there was a reason why they did it this way. Feel free to reopen this if you still have concerns!